src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -4073,7 +4073,7 @@ void M()
     {
         int a = 1;
         int {|Rename:y1|} = C.y;
-        var t = (a, x: y1);
+        var t = (a: a, x: y1);
```

Comment:
We don't need the option.  But every time i make a feature without an option, people ask me to add one :)